presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1199,7 +1199,7 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
       (int)std::thread::hardware_concurrency(),
       cpuLoadPct,
       cpuLoadPct,
-      pool_ ? pool_->currentBytes() : 0,
+      pool_ ? pool_->usedBytes() : 0,
```

Comment:
What's the diff between currentBytes and usedBytes?

Reply:
currentBytes is deprecated. I will remove from velox later.